src/Microsoft.AspNetCore.SignalR/DynamicClientProxy.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Dynamic;
+
+namespace Microsoft.AspNetCore.SignalR
+{ 
+    public class DynamicClientProxy : DynamicObject
+    {
+        private IClientProxy _clientProxy;
```

Comment:
`readonly`

Pretty much any field which is only set in the constructor should be marked `readonly`, as it optimizes field access better. Even if the object itself can mutate, the reference contained in the field never will, and that's what `readonly` refers to here.